README.md

```diff
@@ -65,52 +66,58 @@ This step should be done in the case that changes have been made to any of the y
 
 If changes need to be made to the bundle spec, a new bundle can be generated with:
 ```shell script
-operator-sdk generate csv --csv-channel alpha --csv-version $NEW_VERSION --default-channel --operator-name windows-machine-config-operator --update-crds --from-version $PREV_VERSION
+operator-sdk generate csv --csv-version $NEW_VERSION --operator-name windows-machine-config-operator --update-crds --make-manifests=false
 ```
 
-You should replace `$NEW_VERSION` and `$PREV_VERSION` with the new semver and the previous semver respectively.
+You should replace `$NEW_VERSION` with the new semver.
 This will create a new directory: `deploy/olm-catalog/windows-machine-config-operator/$NEW_VERSION`
 You should use this new directory when [creating the bundle image](#creating-a-bundle-image)
 
 ### Creating a bundle image
-A bundle image can be created by editing the CSV in deploy/bundle/windows-machine-config-operator/manifests/
+A bundle image can be created by editing the CSV in deploy/bundle/windows-machine-config-operator/$NEW_VERSION/
 and replacing `REPLACE_IMAGE` with the location of the WMCO operator image you wish to deploy.
 See [the build instructions](#build) for more information on building the image.
 
 You can then run the following command in the root of this git repository
 ```shell script
-operator-sdk bundle create $BUNDLE_REPOSITORY:$VERSION_TAG -d deploy/olm-catalog/windows-machine-config-operator/0.0.0 \
---channels alpha --default-channel alpha --image-builder podman
```

Comment:
Above you write ` - Instructions assume that the user is using [Podman](https://podman.io/) container engine`, however you are removing the podman specific parts of the command.

Reply:
this is because, we are assuming that user uses `podman` by default. in that case, `--image-builder` option is redundant

Reply:
This command does not work without the --image-builder option if docker is not installed

Reply:
right. 
including this in readme to clear the confusion 
```
Podman can be made default 
by setting an alias for docker as `alias docker='podman'` in `.bashrc`.
```